packages/theming/apple-theme/src/appleTheme.macos.ts

```diff
@@ -46,6 +46,76 @@ export const appleComponents = {
       variant: 'bodyStandard',
     },
   },
+  RadioButton: {
+    tokens: {
+      borderColor: 'rgba(0, 0, 0, 0.15)',
+      backgroundColor: 'white',
+    },
+    root: {
+      accessible: false,
+      focusable: false,
+      style: {
+        alignItems: 'flex-start',
+      },
+    },
+    button: {
```

Comment:
We don't need to add shadow-tops like shadowRadius to match the macOS system control?